### DISCUSSION TITLE
FHRP access permission

### DISCUSSION THREAD

**Discussion by tfboy** [netbox-community/netbox#11455] [Q&A] [answered]
After struggling to figure out how to manage VIPs that move between interfaces, I see there's support for FHRP. Great! Except I don't have it.
I've looked through all the options, and even logged in as a super admin, it's not there.
If I go to /ipam/fhrp-groups/ I get an access denied.
What am I missing?
Thanks!

Sorry, edited to add v3.3.5.
Screenshot:
![image](https://user-images.githubusercontent.com/6972726/211591532-ad138ccd-de4b-4eb8-9dc7-ea3b2a489ebb.png)

**Comment by tfboy** [comment]
Looking at the demo site, I see where it is. In my instance, in the menu under IPAM / Other, it's not listed.
![image](https://user-images.githubusercontent.com/6972726/211592507-5a310672-57c2-4917-bd19-280c40d8653d.png)

**Comment by mtinberg** [comment, marked as answer]
You need to add permissions to your user/group in the Django Administration UI, in the profile in the upper right under Admin then Permissions on the left.  You can use Groups if you want as well.  On upgrade if there are any new permissions (for new models commonly) the upgrade script does _not_ change the permissions tables, so you need to manually choose and add how new features are presented to existing users/groups.

This can be inconvenient but it's probably better than the alternative where new features are presented to anyone until you lock them down, because the upgrade script just isn't going to know what you want to be able to do something sensible in most cases.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: tfboy ***@***.***>
Sent: Tuesday, January 10, 2023 9:20 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] FHRP access permission (Discussion #11455)


After struggling to figure out how to manage VIPs that move between interfaces, I see there's support for FHRP. Great! Except I don't have it.
I've looked through all the options, and even logged in as a super admin, it's not there.
If I go to /ipam/fhrp-groups/ I get an access denied.
What am I missing?
Thanks!

**Reply by tfboy** [reply]
Thank you Mark. I thought it was a permission thing and searched but initially couldn't find anything. But I've now located it and have added it to the necessary groups. :)
